frontend/src/__tests__/cypress/cypress/pages/acceleratorProfile.ts

```diff
@@ -0,0 +1,242 @@
+          ? [...cellValues].sort()
+          : [...cellValues].sort((a, b) =>
+              b === undefined || a === undefined ? -1 : b.localeCompare(a),
+            );
+      expect(cellValues).to.deep.equal(sortedCellValues);
+    });
+    return this;
+  }
+
+  private wait() {
+    this.findAppPage();
+    cy.testA11y();
+  }
+
+  findAppPage() {
+    return cy.findByTestId('app-page-title');
+  }
+
+  findAddButton() {
+    return cy.findByTestId('display-accelerator-modal-button');
+  }
+
+  findEmptyText() {
+    return cy.findByTestId('no-available-accelerator-profiles');
+  }
+
+  findCreateButton() {
+    return cy.findByTestId('create-accelerator-profile');
+  }
+
+  findTableHeaderButton(name: string) {
+    return this.findTable().find('thead').findByRole('button', { name });
+  }
+
+  findResetButton() {
+    return cy.findByRole('button', { name: 'Reset' });
+  }
+
+  findFilterMenuOption() {
+    return cy.findByTestId('filter-dropdown-select');
+  }
+
+  findSearchInput() {
+    return cy.findByRole('textbox', { name: 'Search input' });
+  }
+
+  private findTable() {
+    return cy.findByTestId('accelerator-profile-table');
+  }
+
+  findColumn(column: string) {
+    return this.findTable().find('tbody').find(`[data-label=${column}]`);
+  }
+
+  getRow(column: string, name: string) {
+    return new AcceleratorRow(() => this.findColumn(column).contains(name).parents('tr'));
+  }
+}
+
+class ManageAcceleratorProfile {
+  findAcceleratorNameInput() {
+    return cy.findByTestId('accelerator-name-input');
+  }
+
+  findIdentifierInput() {
+    return cy.findByTestId('accelerator-identifier-input');
+  }
+
+  shouldHaveModalEmptyState() {
+    cy.findByTestId('tolerations-modal-empty-state').should('exist');
+    return this;
+  }
+
+  findTolerationsButton() {
+    return cy.findByTestId('add-toleration-button');
+  }
+
+  findDescriptionInput() {
+    return cy.findByTestId('accelerator-description-input');
+  }
+
+  findSubmitButton() {
+    return cy.findByTestId('accelerator-profile-create-button');
+  }
+
+  findTable() {
+    return cy.findByTestId('toleration-table');
+  }
+
+  getRow(name: string) {
+    return new TolerationRow(() =>
+      this.findTable().find(`[data-label=Key]`).contains(name).parents('tr'),
+    );
+  }
+}
+
+class TolerationsModal extends Modal {
+  constructor(edit = false) {
+    super(edit ? 'Edit toleration' : 'Add toleration');
+  }
+
+  findTolerationKeyInput() {
+    return this.find().findByTestId('toleration-key-input');
+  }
+
+  findTolerationValueAlert() {
+    return this.find().findByTestId('toleration-value-alert');
+  }
+
+  findTolerationSecondRadioCustom() {
+    return this.find().findByTestId('toleration-seconds-radio-custom');
+  }
+
+  findTolerationSecondAlert() {
+    return this.find().findByTestId('toleration-seconds-alert');
+  }
+
+  findTolerationValueInput() {
+    return this.find().findByTestId('toleration-value-input');
+  }
+
+  getTolerationOperatorSelect() {
```

Comment:
```suggestion
  findTolerationOperatorSelect() {
```
Just a command chaining so `find` prefix would be better

Reply:
@pnaik1 whenever we return a cypress selector, the function should use the prefix of `find`. Think of it as finding an element on the page.

`get` should be used for returning a new page object eg.
```
getSection() { 
  return new Section(...);
}
```

Furthermore you can also make functions `private` if it's to only be used internally by this object.